examples/src/utils/Image360UI.ts

```diff
@@ -32,7 +32,7 @@ export class Image360UI {
     const params = {
       siteId: '',
       add: add360ImageSet,
-      premultipliedRotation: true,
+      premultipliedRotation: false,
```

Comment:
Turns out that there are very few datasets that acutally has premultiplied rotation. Only reason it is default true in Reveal is because it is default true in Remote (as in if you don't add this in the Twin Config it assumes true).